Source/VisualStudioTools/Private/VisualStudioToolsCommandlet.cpp

```diff
@@ -185,21 +185,21 @@ struct AssetIndex
 				Classes.Add(OwnerName).Class = Owner;
 			}
 
-			ClassEntry& entry = Classes[OwnerName];
+			FClassEntry& Entry = Classes[OwnerName];
```

Comment:
Which rule is capitalizing Entry?

Reply:
The one in line 26 of the .editorconfig. 

`general_names.style.capitalization` = pascal_case
`general_names.symbols.applicable_kinds` = *

All identifier names are pascal cased by the UE code guidelines. Not something I personally find pretty, but we don't make up the rules :)

Reply:
Totally my bad, I saw pascal case and for some reason mentally mapped it to camel case...